webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -3123,18 +3123,18 @@ def correct_session_data_without_registration_method(self):
         yield correct_session_data_without_registration_method
 
     @pytest.fixture
-    def correct_session_data_with_registration_method_not_none(self, correct_session_data_without_registration_method):
-        correct_session_data_with_registration_method_not_none = {
+    def correct_session_data_with_registration_method_software(self, correct_session_data_without_registration_method):
+        correct_session_data_with_registration_method_software = {
             **correct_session_data_without_registration_method,
             **{'elster_registration_method_eligibility': 'software'}}
-        yield correct_session_data_with_registration_method_not_none
+        yield correct_session_data_with_registration_method_software
```

Comment:
Just because Niko [pointed that out in my PR](https://github.com/digitalservice4germany/steuerlotse/pull/221#discussion_r725967850) yesterday:
>Why yield instead of return? Should be exactly the same. You're not doing anything after yielding (like teardown/cleanup steps)...

 I think we could use return here because we do nothing after yielding.